data/priyanshu096.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Priyanshu Patel",
+  "type": "personal",
+  "bio": "Web Developer **Esports Player**. Travel enthusiast ",
+  
+   
+  
+
+"links": [
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/priyans.121/",
+      "icon": "FaInstagram"
+    } ,
+ {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Priyans121",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on Linkdin",
```

Comment:
There is a minor typo in the spelling of **' Linkedin '** 
You can correct it :+1: